notdiamond/toolkit/rag/llms.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
```

Comment:
Can you delete these `env` statements from these modules? They are not executed as scripts.

```suggestion
```